### PR TITLE
Adds Space Ruin: Voidhopper of Nexus

### PR DESCRIPTION
**Pull request by haydenredacted**
## What Does This PR Do

This PR adds a new space ruin called Voidhopper of Nexus (freighter.dmm). This ruin is a cargo ship with a variety of different items such as raw materials and mech equipment, it also comes with ~~Russian~~ Soviet Nians, a new simple mob that acts similar to the ~~Russians~~ Soviets you can find in space.

The shipping manifest of the cargo ship is as follows:
1 pair of kitty ears 
1 banhammer 
1 exosuit mining scanner 
1 diamond-tipped exosuit drill 
1 'LittleBoy' Mining Grenade Launcher 
1 mecha voice modification kit: Honk 
20 plastitanium glass 
50 uranium sheets 
50 reinforced plasma glass 
50 metal sheets 
1 fuel tank 
1 radiant dance machine mark IV 
1 soap 
1 advanced mop 
1 advanced space cleaner 

## Why It's Good For The Game

More space ruins makes exploring space more fun.
Who said ~~space Russians~~ Soviets could only be human?

## Images of changes
StrongDMM view:
image
~~Russian~~ Soviet Nian:
image

## Testing

Loaded up a server, found the ruin and all its loot. Killed the ~~Russian~~ Soviet Nians, the corpse with all the loot it should have dropped properly.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
add: Added a new space ruin, Voidhopper of Nexus
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
I would expand the engine room for flavour.

Whilst asymmetrical ships are cool, the the centerline of the ship feels off. I'd lean harder into the asymmetry if you're going to employ it.

File: _maps/map_files/RandomRuins/SpaceRuins/freighter.dmm [hunk 1]
```diff
@@ -0,0 +1,1261 @@
...
+	},
+/obj/effect/spawner/random_spawners/blood_maybe,
+/turf/simulated/floor/plasteel{
+	icon_state = "dark"
+	},
+/area/ruin/space/powered)
+"XB" = (
+/obj/structure/table,
+/obj/item/paper{
+	info = "<h3>Voidhopper of Nexus Shipping Manifest</h3><hr><br>Order: #7934<br>Destination: <i> The destination seems to have been blacked out.</i><br>Requested By: Athena, Praetorian of Gliese<br>CONTENTS:<br><ul><li>1 pair of kitty ears</li><li>1 banhammer</li><li>1 exosuit mining scanner</li><li>1 diamond-tipped exosuit drill</li><li>1 "LittleBoy" Mining Grenade Launcher</li><li>1 mecha voice modification kit: Honk</li><li>20 plastitanium glass</li><li>50 uranium sheets</li><li>50 reinforced plasma glass</li><li>50 metal sheets</li><li>1 fuel tank</li><li>1 radiant dance machine mark IV</li><li>1 soap</li><li>1 advanced mop</li><li>1 advanced space cleaner</li></ul><br>CHECK CONTENTS AND SIGN BELOW THE LINE TO CONFIRM RECEIPT OF GOODS<hr><b>X: <i> Athena, Praetorian of Gliese </i> </b>";
```
Comment (warriorstar-orion) [inline _maps/map_files/RandomRuins/SpaceRuins/freighter.dmm:642]: create a file `/code/modules/away_missions/mission_code/ruins/nian_freighter.dm` or similar and put this in there as a paper subtype:

```c
/obj/item/paper/fluff/ruins/nian_freighter/manifest
	name = "Voidhopper of Nexus Shipping Manifest"
	info = {"
		<h3>Voidhopper of Nexus Shipping Manifest</h3>
		<hr />
		<br />Order: #7934<br />Destination:
		<i> The destination seems to have been blacked out.</i><br />Requested By:
		Athena, Praetorian of Gliese<br />CONTENTS:<br />
		<ul>
		  <li>1 pair of kitty ears</li>
		  <li>1 banhammer</li>
		  <li>1 exosuit mining scanner</li>
		  <li>1 diamond-tipped exosuit drill</li>
		  <li>1 "LittleBoy" Mining Grenade Launcher</li>
		  <li>1 mecha voice modification kit: Honk</li>
		  <li>20 plastitanium glass</li>
		  <li>50 uranium sheets</li>
		  <li>50 reinforced plasma glass</li>
		  <li>50 metal sheets</li>
		  <li>1 fuel tank</li>
		  <li>1 radiant dance machine mark IV</li>
		  <li>1 soap</li>
		  <li>1 advanced mop</li>
		  <li>1 advanced space cleaner</li>
		</ul>
		<br />CHECK CONTENTS AND SIGN BELOW THE LINE TO CONFIRM RECEIPT OF GOODS
		<hr />
		<b>X: <i> Athena, Praetorian of Gliese </i> </b>
	"}
```

See the similar ruin papers in `code\modules\awaymissions\mission_code\ruins\deepstorage.dm` as examples.

Comment (haydenredacted) [reply _maps/map_files/RandomRuins/SpaceRuins/freighter.dmm]: Alright, could you check if I did what you said to do properly? Thanks!

**Comment by Coolrune206** [comment]
Why are there cat ears in there

**Comment by haydenredacted** [comment]
There's cat ears because someone ordered them!

**Comment by Denthamos** [comment]
When adding new ruins like this you need to detail what loot is to be found in them in the PR's description.

**Comment by haydenredacted** [comment]
added the list of loot available

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
Lamp on window

2024_10_04__03_53_02__paradise dme  freighter dmm  - StrongDMM

The engine heaters have a noticeable gap between them and the wall. Other shuttles usually place a reinforced window there to obscure it:

2024_10_04__03_57_12__paradise dme  freighter dmm  - StrongDMM


2024_10_04__03_54_43__paradise dme  emergency_cyb dmm  - StrongDMM

The survival pod windows here might look good and fit well with the plastitanium theme:

2024_10_04__03_56_42__paradise dme  freighter dmm  - StrongDMM

The stair railings should probably have end caps on them, same for the ones on the bridge.

2024_10_04__03_58_51__paradise dme  freighter dmm  - StrongDMM

2024_10_04__03_59_02__paradise dme  freighter dmm  - StrongDMM

File: paradise.dme [hunk 2]
```diff
@@ -788,8 +788,8 @@
 #include "code\game\gamemodes\traitor\traitor.dm"
 #include "code\game\gamemodes\trifecta\trifecta.dm"
 #include "code\game\gamemodes\vampire\traitor_vamp.dm"
-#include "code\game\gamemodes\vampire\vampire_gamemode.dm"
 #include "code\game\gamemodes\vampire\vampire_chan.dm"
+#include "code\game\gamemodes\vampire\vampire_gamemode.dm"
```
[the remark this answers is not in the document]
Comment (haydenredacted) [reply paradise.dme]: is there an easy way to revert this? it kinda just happened on its own and I'm not sure what I did to have caused it.

File: code/modules/mob/living/simple_animal/hostile/russian.dm [hunk 3]
```diff
@@ -1,11 +1,11 @@
-/mob/living/simple_animal/hostile/russian
-	name = "Russian"
-	desc = "For the Motherland!"
+/mob/living/simple_animal/hostile/soviet
+	name = "Soviet"
+	desc = "For the Union!"
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/hostile/russian.dm:3]: You have to update this PR with the latest changes that master had, no need to do manual stuff like this.

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 4]
```diff
@@ -114,6 +114,22 @@
 	name = "Corpse of a Ranged Russian"
 	head = /obj/item/clothing/head/ushanka
 
+/obj/effect/mob_spawn/human/corpse/sovietnian
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/corpse.dm:117]: change to use snake_case (soviet_nian) here and in other places

**Comment by Henri215** [comment]
Ask for help over the discord if you are having difficulties updating your master.

**Review by AffectedArc07 — APPROVED** [review]
Config change added to prod

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
The worme won't spawn in the cabinet because only objects are inserted into closets, not mobs:

2024_10_28__03_10_18__Paradise Station 13

You'd have to update this code to get that to work:

https://github.com/ParadiseSS13/Paradise/blob/21321ca8cdd6db6c4653a1b64858b208e6eff492/code/game/objects/structures/crates_lockers/closets.dm#L151-L159

You'll want to avoid putting mobs near transparent windows because they will aggro and try and escape to attack mobs outside it.

2024_10_28__03_17_50__Paradise Station 13

The windows you added to the rear thrusters are missing from the side thrusters:

2024_10_28__11_22_27__paradise dme  freighter dmm  - StrongDMM
2024_10_28__11_22_31__paradise dme  freighter dmm  - StrongDMM

**Review by warriorstar-orion — APPROVED** [review]
LGTM

**Comment by DGamerL** [comment]
This PR is going stale, please deconflict!

File: code/datums/ruins/space_ruins.dm [hunk 5]
```diff
@@ -370,6 +370,11 @@
 	description = "A security shuttle that has been floating in space."
 	allow_duplicates = FALSE
 
+/datum/map_template/ruin/space/freighter
+	id = "freighter"
+	suffix = "freighter.dmm"
+	name = "Voidhopper of Nexus"
+	description = "A cargo ship headed to a nearby system."
```
Comment (Burzah) [inline code/datums/ruins/space_ruins.dm:377]:
```suggestion
	description = "A cargo ship headed to a nearby system."
	
```

**Review by Burzah — APPROVED** [review]

**Pull request merged**